code/ui/manager/src/components/sidebar/Heading.tsx

```diff
@@ -87,6 +90,9 @@ export const Heading: FC<HeadingProps & ComponentProps<typeof HeadingWrapper>> =
         <Brand />
       </BrandArea>
 
+      {extra.map(({ id, render: Render }) => (
```

Comment:
Should either this be `extra?.map`, or the prop at the top have a default value `extra = []`? The stories are failing for this reason, you can also just fix them in their files

Reply:
fixed